### PR TITLE
Update for new flowcollecto and other changes

### PR DESCRIPTION
**Pull request by memodi**
- reflects recent changes in the Operator.
- update catalog image location.

### CODE REVIEW

**Comment by memodi** [comment]
Here's the success run: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/memodi-e2e-benchmarking-multibranch-pipeline/job/update_flowcollector/5/console

File: Jenkinsfile [hunk 1]
```diff
@@ -78,6 +78,7 @@ pipeline {
             name: 'INSTALLATION_SOURCE',
             choices: ['OperatorHub', 'Source', 'None'],
             description: '''
+                Select Source until 0.1.5 NOO release since flowcollector has breaking changes<br/>
```
Comment (nathan-weinberg) [inline Jenkinsfile:81]:
```suggestion
                <b>Select Source until 0.1.5 NOO release since flowcollector has breaking changes</b><br/>
```

Comment (nathan-weinberg) [reply Jenkinsfile:81]: One small request, otherwise LGTM

**Pull request merged**